util/gh-pages/script.js

```diff
@@ -579,6 +579,32 @@ function setTheme(theme, store) {
     }
 }
 
+function handleShortcut(ev) {
+    if (ev.ctrlKey || ev.altKey || ev.metaKey || disableShortcuts) {
```

Comment:
Where's `disableShortcuts` from? I don't see it in the rest of the script

Reply:
From my tired brain. ^^'

Removed it!